readme.md

```diff
@@ -58,6 +58,10 @@ This list is mainly about [CSS](https://developer.mozilla.org/docs/Web/CSS) –
 
 ## Layout
 
+- [Interactive Guide to CSS Container Queries by Ahmad Shadeed](https://ishadeed.com/article/container-queries/) - Interactive guide with practical examples.
+- [Container Queries: A Practical Introduction by Miriam Suzanne](https://www.oddbird.net/2021/04/05/containerqueries/) - Technical and practical introduction.
+- [CSS Container Queries Explained by Kevin Powell](https://www.youtube.com/watch?v=5J3TK7bPuSg) - Clear video tutorial with real-world examples.
+
```

Comment:
Please remove the changes that are not related to Houdini from this PR. Fell free to force push to your fork/branch.